Announcement for Quarkus 2.0.0.Final (#1037)
quarkusio/quarkusio.github.io: _posts/2021-06-30-quarkus-2-0-0-final-released.adoc

---
layout: post
title: 'Quarkus 2.0.0.Final released - Vert.x 4, MicroProfile 4, Continuous Testing and much more'
date: 2021-06-30
tags: release

> synopsis: 'You were waiting for it and it is finally here: Quarkus 2.0.0.Final with all its bells and whistles.'

Reviewer: ```suggestion
synopsis: 'You have been waiting for it and it is finally here: Quarkus 2.0.0.Final with all its bells and whistles.'
```

author: gsmet
---

The Quarkus team is proud to announce the availability of the Final release of Quarkus 2.0.

This version has been a gigantic effort to bring Quarkus to a whole new level, while keeping its roots:
fast boot, low memory usage and developer joy.

A big thank you to everyone involved in the community, contributors, testers...
This version is a team effort and wouldn't be as great without all of you.

> We already presented the novelties of this version in the Alphas/CRs announcements but here is a summary of what's new in this version.

Reviewer: This seems a tad out of place, maybe move it into `What's New`?

as usual, we have a https://github.com/quarkusio/quarkus/wiki/Migration-Guide-2.0[migration guide] to help you.

If you encounter any issue upgrading to this new version, please report it so that we can either fix it or add the information to the guide.

== What's new?

=== JDK 11 as minimal version

JDK 11 is the minimal version to use Quarkus 2.0.

JDK 8 is not supported anymore (but you can keep using 1.13.x if you are stuck with it!).

=== GraalVM 21.1

GraalVM 21.1 is the recommended version for Quarkus 2.0.

=== Vert.x 4

Vert.x is at the core of Quarkus.

Quarkus 2.0 brings Vert.x 4.
You can learn more about it in https://vertx.io/blog/eclipse-vert-x-4-released/[the announcement] and the https://vertx.io/blog/whats-new-in-vert-x-4/[What's new blog post].

=== Eclipse MicroProfile 4

MicroProfile 4 is the latest evolution of the https://github.com/eclipse/microprofile/releases/tag/4.0[MicroProfile specifications].

All the SmallRye components in Quarkus have been updated to the latest and greatest and, apart from implementing MicroProfile 4,
they also come with additional new features.

=== Continuous Testing

> I'll let the stage to Stuart Douglas for that one as he's the author of this amazing new feature.

Reviewer: ```suggestion
In the words of Stuart Douglas (as he's the author of this amazing new feature):
```
